addons/scratchstats/userscript.js

```diff
@@ -0,0 +1,115 @@
+function createItem(number, label) {
+  const item = document.createElement("div");
+  const numberDiv = document.createElement("div");
+  item.appendChild(numberDiv);
+  numberDiv.className = "sa-stats-number";
+  numberDiv.innerText = number;
+  const labelDiv = document.createElement("div");
+  item.appendChild(labelDiv);
+  labelDiv.innerText = label;
+  return item;
+}
+
+export default async function ({ addon, msg, console }) {
+  const username = location.pathname.split("/")[2];
+  if (!username) return;
+  const content = document.querySelector("#content");
+  const commentBox = document.querySelector("#content > .box:not(#profile-data):not(.slider-carousel-container)");
+  const statsBox = document.createElement("div");
+  content.insertBefore(statsBox, commentBox);
+  statsBox.className = "box sa-stats slider-carousel-container";
+  const statsHeader = document.createElement("div");
+  statsBox.appendChild(statsHeader);
+  statsHeader.className = "box-head";
+  const statsTitle = document.createElement("h4");
+  statsHeader.appendChild(statsTitle);
+  statsTitle.innerText = msg("title");
+  const statsMoreLink = document.createElement("a");
+  statsHeader.appendChild(statsMoreLink);
+  statsMoreLink.innerText = msg("view-more");
+  statsMoreLink.href = "https://scratchstats.com/" + username;
+  const statsMoreIcon = document.createElement("img");
+  statsMoreLink.insertBefore(statsMoreIcon, statsMoreLink.firstChild);
+  statsMoreIcon.src = addon.self.dir + "/scratchstats.png";
+  const stats = document.createElement("div");
+  statsBox.appendChild(stats);
+  stats.className = "box-content";
+  stats.innerText = msg("loading");
+
+  fetch(`https://scratchdb.lefty.one/v2/user/info/${username}`).then(async function (response) {
+    stats.removeChild(stats.firstChild); // remove loading message
+    const followRow = document.createElement("div");
+    stats.appendChild(followRow);
+    followRow.className = "sa-stats-row";
+    const ranksRow = document.createElement("div");
+    stats.appendChild(ranksRow);
+    ranksRow.className = "sa-stats-row";
+    const data = await response.json();
+    followRow.appendChild(createItem(data.statistics.followers, msg("followers")));
```

Comment:
Maybe we add a thousand comma separator (maybe in all numbers, but most importantly on followers?)

Reply:
Probably worth it for ranking numbers as well, as some users get 6 digit numbers there

Reply:
We could alternatively go with `6K`, `7M`, etc